### DISCUSSION TITLE
"Blood Runecrafting" Tracker Overlay

### DISCUSSION THREAD

**Discussion by ivydoomsday** [runelite/runelite#17046] [Q&A] [answered]
This overlay randomly popped up on my screen, it does not have a shift right click delete infobox option and ive turned off every single runecrafting plugin there is and its still there. restarting client makes it go away til i teleport and it is irritating especially when im only 68 rc

![image](https://github.com/runelite/runelite/assets/145387652/481204b8-05af-4326-9bf0-f7ed7d5bc5ec)

**Comment by geheur** [comment, marked as answer]
zeah rc tracker

**Reply by ivydoomsday** [reply]
I LOVE YOU
